Tests/TensorFlowTests/TensorAutoDiffTests.swift

```diff
@@ -575,16 +567,6 @@ final class TensorAutoDiffTests: XCTestCase {
         XCTAssertEqual(pb(Tensor(ones: [3, 3])), Tensor(repeating: 5.9604645e-08, shape: [3, 3]))
     }
 
-    // SR-9345
```

Comment:
QQ: What happened to these tests? Any reason why they were deleted?

Reply:
I decided to delete the test because it's no longer meaningful: it was added in https://github.com/apple/swift/pull/20788, but differentiation implementation details have significantly changed since. The test is also not inherently tied to `Tensor` - the AutoDiff test suite at `apple/swift` covers such tests.

The test calls a top-level function `foo` with a `@differentiable(vjp vjpFoo:)` attribute. It's possible to remove the attribute and add `@derivative(of: foo)` to `func vjpFoo`, there's no issue there.

However, defining a vague top-level function `foo` isn't nice. Moving `func foo` and `func vjpFoo` to a local context would be better, but isn't possible because [`@derivative` is disabled in local contexts](https://github.com/apple/swift/pull/28483).